cloudslang-tests/src/test/java/io/cloudslang/lang/systemtests/SimpleFlowTest.java

```diff
@@ -50,7 +50,7 @@ public void testSimpleFlowBasic() throws Exception {
     }
 
     @Test(timeout = DEFAULT_TIMEOUT)
-    public void testSimpleFlowBasicOneLinerSyntax() throws Exception {
+    public void testSimpleFlowInvalidOneLinerSyntax() throws Exception {
```

Comment:
Are you trying to test here that one-liner syntax is no more valid? (we should have a test for that)
maybe change the name to `testOneLinerIsInvalid`? WDYT?